webui/tests/selenium/webui-selenium-test.py

```
@@ -234,48 +234,55 @@ def test_client_disabling(self):
         self.login()
-        # Clicks on client menue tab
-        self.select_navbar_element("client")
-        # Tries to click on client...
+        # Navigate to client list and verify client is reachable
+        self.driver.get(self.base_url + "/client")
+        self.wait_for_spinner_absence()
         try:
             self.wait_and_click(By.LINK_TEXT, self.client)
         # Raises exception if client not found
         except ElementTimeoutException:
             raise ClientNotFoundException(self.client)
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)